rotary_irq_esp.py

```diff
@@ -12,17 +12,17 @@
 from rotary import Rotary
 from sys import platform
 
-_esp8266_blacklist_pins = [16]
+_esp8266_deny_pins = [16]
```

Comment:
Thanks for improving the error message and changing to a racial-neutral implementation.  I did some googling on alternatives to blacklist/whitelist.  The terms blocklist/allowlist are emerging as the top contenders.  e.g.  Google Chromium project.  Please consider changing `deny` to `blocklist`.  If you think `deny `is better I'll accept the PR as-is.  Your choice.  Thank you again for this contribution !

Reply:
I went with deny because of what I saw from the Twitter engineering team, but I'm not attached to it

Reply:
I'm not sure how to make changes to the PR as it is, but if you propose changes I'm fine with that

Reply:
> from the Twitter engineering team,

sounds good.  Let's stay with `deny`